iati-activities-schema.xsd

```diff
@@ -2867,5 +2579,269 @@
     </xsd:annotation>
   </xsd:attribute>
 
+<!-- locationBase provides only the ref attribute for location -->
+  <xsd:complexType name="locationBase">
+    <xsd:sequence>
+      <xsd:any namespace="##other" processContents="lax" minOccurs="0" maxOccurs="unbounded"/>
+    </xsd:sequence>
+    <xsd:attribute name="ref" use="optional" type="xsd:string">
+      <xsd:annotation>
+        <xsd:documentation xml:lang="en">
+          A cross-reference to the internal reference assigned to a defined location: iati-activity/location/\@ref.
+        </xsd:documentation>
+      </xsd:annotation>
+    </xsd:attribute>
+    <xsd:anyAttribute processContents="lax" namespace="##other"/>
+  </xsd:complexType>
+
+<!-- locationFull extends locationBase to include all sub-elements for location at the iati-activity level -->
+  <xsd:complexType name="locationFull">
+    <xsd:complexContent>
+      <xsd:extension base="locationBase">
+        <xsd:sequence>
+          <xsd:element name="location-reach" minOccurs="0" maxOccurs="1">
+            <xsd:annotation>
+              <xsd:documentation xml:lang="en">
+                 Does this location describe where the activity takes
```

Comment:
Dangit - the hard wrapping is here too. 

I'd suggest any new lines added should not have hard wrapping. Once this is fixed for this and new lines below, it's good to go!

Reply:
I have nothing good to say about the irritation this removing hard wrapping from old data is causing.